src/config/config.go

```diff
@@ -1,7 +1,145 @@
 package config
 
-const InsertDefaultRank bool = true
-const LogDumpLocation string = "logdump/"
+import (
+	"os"
+	"strings"
 
-// update this when loading config
-var NumberOfEngines int = 10
+	"github.com/knadh/koanf/parsers/yaml"
+	"github.com/knadh/koanf/providers/env"
+	"github.com/knadh/koanf/providers/file"
+	"github.com/knadh/koanf/providers/structs"
+	"github.com/knadh/koanf/v2"
+	"github.com/rs/zerolog/log"
+	"github.com/tminaorg/brzaguza/src/structures"
+)
+
+type SESettings struct {
+	Timeout  int                 `koanf:"timeout"`
+	Shortcut string              `koanf:"shortcut"`
+	Crawlers []structures.Engine `koanf:"crawlers"`
+}
```

Comment:
What does Crawlers represent? I read the default settings and still can't get a good idea on whats the idea behind it.

Reply:
Also, a comment about the unit of time for Timeout would be useful.

Reply:
Crawlers used for each SE, info was gathered from here: https://www.searchenginemap.com/

Could be renamed to `Sources`